ruff.toml

```diff
@@ -14,3 +14,7 @@ select = [
 # Rule configuration.
 line-length = 120
 
+[extend-per-file-ignores]
+"templates/shift-assignment-ortools/main.py" = ["C901"]
+"templates/shift-planning-ortools/main.py" = ["C901"]
+"templates/shift-assignment-pyomo/main.py" = ["C901"]
```

Comment:
I should have started using ruff for that tech talk as well. Are these easy fixes? :thinking: 
Probably nothing we want to spend time on though.

Reply:
I'd leave it for not. I fixes a couple of other things